target/linux/armsr/modules.mk

```diff
@@ -287,6 +287,23 @@ define KernelPackage/octeontx2-net
 endef
 $(eval $(call KernelPackage,octeontx2-net))
 
+define KernelPackage/renesas-net-avb
+SUBMENU:=$(NETWORK_DEVICES_MENU)
```

Comment:
Please use an indention of `  `. for this block.

Reply:
Hi @hauke 
Thanks. I've updated this.